src/main/java/algorithms/ui/SemanticMatcher.java

```diff
@@ -37,8 +37,10 @@
 public class SemanticMatcher {
 	
 	
-	static File ontoFile1 = new File("./files/_PHD_EVALUATION/BIBFRAME-SCHEMAORG/ONTOLOGIES/bibframe.rdf");
-	static File ontoFile2 = new File("./files/_PHD_EVALUATION/BIBFRAME-SCHEMAORG/ONTOLOGIES/schema-org.owl");
+	//static File ontoFile1 = new File("./files/_PHD_EVALUATION/BIBFRAME-SCHEMAORG/ONTOLOGIES/bibframe.rdf");
+	//static File ontoFile2 = new File("./files/_PHD_EVALUATION/BIBFRAME-SCHEMAORG/ONTOLOGIES/schema-org.owl");
```

Comment:
Do we need these commented out? Would it be enough to just straight up change them?

Reply:
It's easier to switch between the ontoFiles if the unused ones are commented, and this class is mostly used for testing if something works correctly, since it's not used in requests.